spec/RecapSpec.js

```diff
@@ -133,6 +133,42 @@ describe('The Recap export module', function() {
     });
   });
 
+  describe('uploadDocket, uploadAttachmentMenu, uploadDocument', function() {
+    var existingFormData;
+    beforeEach(function() {
+      setupChromeSpy();
+      existingFormData = window.FormData;
+      window.FormData = FormDataFake;
+    });
+    
+    afterEach(function() {
+      window.FormData = existingFormData;
+      removeChromeSpy();
+    });
+
+    var bytes = new Uint8Array([100, 100, 200, 200, 300]);
+
+    it('all send debug=false in AJAX request', function() {
```

Comment:
Can we be more expository? E.g.

```patch
diff --git a/spec/RecapSpec.js b/spec/RecapSpec.js
index e3933d0..e092f36 100644
--- a/spec/RecapSpec.js
+++ b/spec/RecapSpec.js
@@ -148,7 +148,7 @@ describe('The Recap export module', function() {
 
     var bytes = new Uint8Array([100, 100, 200, 200, 300]);
 
-    it('all send debug=false in AJAX request', function() {
+    it('all send debug=false in AJAX request.\nThis probably means debug=TRUE is set in recap.js:4.', function() {
       expected = new FormDataFake();
       expected.append('debug', false);
```

So that the result is a little more helpful:

```
pb3:recap-chrome jhawk$ karma start --single-run --browsers Chrome_CI
20 03 2018 20:39:04.703:INFO [karma]: Karma v2.0.0 server started at http://0.0.0.0:9876/
20 03 2018 20:39:04.707:INFO [launcher]: Launching browser Chrome_CI with unlimited concurrency
20 03 2018 20:39:04.711:INFO [launcher]: Starting browser Chrome
20 03 2018 20:39:05.017:INFO [HeadlessChrome 0.0.0 (Mac OS X 10.10.5)]: Connected on socket RKr45QP1n7b7urv7AAAA with id 2307793
HeadlessChrome 0.0.0 (Mac OS X 10.10.5) The Recap export module uploadDocket, uploadAttachmentMenu, uploadDocument all send debug=false in AJAX request.
This probably means debug=TRUE is set in recap.js:4. FAILED
	Expected FormDataFake({ court: 'nysd', pacer_case_id: '437098', upload_type: 1, filepath_local: [object Blob], debug: true }) to equal <jasmine.objectContaining(FormDataFake({ debug: false }))>.
	    at UserContext.<anonymous> (spec/RecapSpec.js:157:26)
	Expected FormDataFake({ court: 'nysd', pacer_case_id: '437098', upload_type: 2, filepath_local: [object Blob], debug: true }) to equal <jasmine.objectContaining(FormDataFake({ debug: false }))>.
	    at UserContext.<anonymous> (spec/RecapSpec.js:161:26)
	Expected FormDataFake({ court: 'nysd', pacer_case_id: '437098', pacer_doc_id: '127015406472', document_number: '4', attachment_number: '1', filepath_local: [object Blob], upload_type: 3, debug: true }) to equal <jasmine.objectContaining(FormDataFake({ debug: false }))>.
	    at UserContext.<anonymous> (spec/RecapSpec.js:167:26)
HeadlessChrome 0.0.0 (Mac OS X 10.10.5): Executed 93 of 93 (1 FAILED) (0.123 secs / 0.118 secs)
pb3:recap-chrome jhawk$ 
```

rather than just:

```
...
HeadlessChrome 0.0.0 (Mac OS X 10.10.5) The Recap export module uploadDocket, uploadAttachmentMenu, uploadDocument all send debug=false in AJAX request FAILED
...
```

Reply:
Done